### PR TITLE
Bulbs quiz transplant

### PR DESCRIPTION
**Pull request by daytonn**
I transplanted the quiz code over here. It's not a refactor. I basically moved the jQuery code from one place to another. While this is not the ideal case and we will want to refactor eventually, it's a starting point for cleaning this up.

### CODE REVIEW

**Comment by collin** [comment]
Makes sense given most of the source is just copied over.

We need to:

- [ ] add $ to globals for eslint
- [ ] verify $ isn't getting compiled into the dist files

File: elements/bulbs-quiz/bulbs-quiz-cms.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {
+  registerElement,
+  EmbededCMSElement,
+} from 'bulbs-elements/register';
+
+class EmbeddedBulbsQuiz extends EmbededCMSElement {
```
Comment (spra85) [inline elements/bulbs-quiz/bulbs-quiz-cms.js]: So minor but we should eventually fix the misspelling of `EmbededCMSElement` to `EmbeddedCMSElement`

File: elements/bulbs-quiz/bulbs-quiz.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+bulbs-quiz {
+  display: block;
```
Comment (spra85) [inline elements/bulbs-quiz/bulbs-quiz.scss]: Wouldn't this be the default?

**Comment by spra85** [comment]
LGTM, should verify the two concerns @collin had

File: elements/bulbs-quiz/bulbs-quiz.test.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe('TestQuiz', () => {
+    it('instantiates a TestQuiz when the quiz style is test', (done) => {
+      fixtureContainer.appendChild(subject);
+      setImmediate(() => {
```
Comment (kand) [inline elements/bulbs-quiz/bulbs-quiz.test.js]: What is `setImmediate` used for in these tests?

**Comment by kand** [comment]
👍

File: elements/bulbs-quiz/bulbs-quiz.scss (hunk 2 above)
Comment (daytonn) [inline elements/bulbs-quiz/bulbs-quiz.scss]: I don't think it is in the case of custom elements. @collin can you confirm this?

**Pull request merged**